wolfcrypt/src/port/iotsafe/iotsafe.c

```diff
@@ -448,12 +476,14 @@ static int iotsafe_readfile(uint8_t file_id, unsigned char *content,
         off_p2 = (off & 0xff);
         iotsafe_cmd_start(csim_cmd, IOTSAFE_CLASS, IOTSAFE_INS_READ_FILE,
                 off_p1, off_p2);
-        iotsafe_cmd_add_tlv(csim_cmd, IOTSAFE_TAG_FILE_ID, 1, &file_id);
+        iotsafe_cmd_add_tlv(csim_cmd, IOTSAFE_TAG_FILE_ID, file_id_sz, file_id);
         iotsafe_cmd_complete(csim_cmd);
         ret = expect_csim_response(csim_cmd, (word32)XSTRLEN(csim_cmd), &resp);
         if (ret > 0) {
+            /*
```

Comment:
Should this code be removed? If not please add comment indicating why it is commented out.